playbook/app/pb_kits/playbook/playbook-rails.js

```diff
@@ -9,6 +9,10 @@ window.datePickerHelper = datePickerHelper
 import dialogHelper from './pb_dialog/dialogHelper'
 window.dialogHelper = dialogHelper
 
+// CopyContent
```

Comment:
@jasperfurniss I'm not sure we want this here. This file is intended for kits not docs.

Reply:
I'll go ahead and move this elsewhere.